test_workspace.bzl

```diff
@@ -8,13 +8,13 @@ def play_routes_compiler_cli_test_repositories():
     maven_install(
         name = "play_routes_compiler_cli_test_maven",
         artifacts = [
-            "org.specs2:specs2-common_2.12:4.7.0",
-            "org.specs2:specs2-core_2.12:4.7.0",
-            "org.specs2:specs2-matcher_2.12:4.7.0",
-            "com.typesafe.akka:akka-actor_2.12:2.5.24",
-            "com.typesafe.play:play_2.12:2.7.4",
-            "com.typesafe.play:play-test_2.12:2.7.4",
-            "com.typesafe.play:play-specs2_2.12:2.7.4",
+            "org.specs2:specs2-common_2.13:4.7.0",
+            "org.specs2:specs2-core_2.13:4.7.0",
+            "org.specs2:specs2-matcher_2.13:4.7.0",
+            "com.typesafe.akka:akka-actor_2.13:2.5.24",
+            "com.typesafe.play:play_2.13:2.7.4",
```

Comment:
should these use the play 2.8.2 versions?

Reply:
ya. I will update the test ones.